.github/workflows/ci.yml

```
@@ -12,6 +12,13 @@ env:
   NODE_OPTIONS: --max-old-space-size=6144
 
 jobs:
+  commitlint:
+    runs-on: ubuntu-latest
```

suggestion: add strict permissions for security concerns
```suggestion
  commitlint:
    runs-on: ubuntu-latest
    permissions:
      actions: read
      contents: read
      # required by wagoid/commitlint-github-action
      pull-requests: read
```

---

@LayZeeDK Done. Thank you for the suggestion.